packages/dashboard/src/app/views/exploreTemplates/index.js

```diff
@@ -81,6 +81,14 @@ function ExploreTemplates() {
     });
   }, [templatesOrderById, templates]);
 
+  const handleCreateStoryFromTemplate = useCallback(
+    (templateId) => {
+      const template = templates[templateId];
+      createStoryFromTemplate(template);
+    },
+    [createStoryFromTemplate, templates]
+  );
```

Comment:
Maybe this seems a little arbitrary/pre-emptive. I want to get to the point where we can just pass the template ID to the api and have the story get generated. I set it up like this so that I could start to decrease the parts of the `template` object we need in exploreTemplates.

Reply:
Sounds good to me 👍 

I would also like to get to this point.

It will become more feasible once we also only display images in the detail view. Then we can move the template data out of the JS bundle so it can be consumed in PHP. Later on, we can consume everything from an external REST API (think a dedicated stories templates website)